tests/beaker_tests/cisco_acl/test_ace.rb

```diff
@@ -13,307 +13,136 @@
-    dst_addr        => 'any',
-  ",
-  resource:       {
-    'action'   => 'deny',
-    'proto'    => 'tcp',
-    'src_addr' => 'any',
-    'dst_addr' => 'any',
+# Test hash test cases
+tests[:seq_5] = {
+  preclean:       'cisco_acl',
+  title_pattern:  'ipv4 beaker 5',
+  manifest_props: {
+    # 'remark' is a standalone property
+    remark: 'seq_5 remark'
   },
 }
 
-tests['ipv4_seq_30'] = {
-  desc:           '1.3 ipv4 beaker_1 seq 30',
-  title_pattern:  'ipv4 beaker_1 30',
-  manifest_props: "
-    action        => 'permit',
-    proto         => 'tcp',
-    src_addr      => '1.2.3.4 2.3.4.5',
-    src_port      => 'eq 40',
-    dst_addr      => '8.9.0.4/32',
-    dst_port      => 'range 32 56',
-    tcp_flags     => 'ack syn fin',
-    precedence    => 'flash',
-    established   => true,
-    packet_length => 'range 80 1000',
-    time_range    => 'my_range',
-    redirect      => 'Ethernet1/1,Ethernet1/2,port-channel1',
-  ",
-  resource:       {
-    'action'        => 'permit',
-    'proto'         => 'tcp',
-    'src_addr'      => '1.2.3.4 2.3.4.5',
-    'src_port'      => 'eq 40',
-    'dst_addr'      => '8.9.0.4/32',
-    'dst_port'      => 'range 32 56',
-    'tcp_flags'     => 'ack syn fin',
-    'precedence'    => 'flash',
-    'established'   => 'true',
-    'packet_length' => 'range 80 1000',
-    'time_range'    => 'my_range',
-    'redirect'      => 'Ethernet1/1,Ethernet1/2,port-channel1',
+tests[:seq_10_v4] = {
+  title_pattern:  'ipv4 beaker 10',
+  manifest_props: {
+    action:   'permit',
+    proto:    'tcp',
+    src_addr: '1.2.3.4 2.3.4.5',
+    dst_addr: '9.9.0.4/32',
   },
 }
 
-tests['ipv6_seq_10'] = {
-  desc:           '2.1 ipv6 beaker_6 seq 10',
-  title_pattern:  'ipv6 beaker_6 10',
-  manifest_props: "
-    action          => 'deny',
-    proto           => 'tcp',
-    src_addr        => 'any',
-    dst_addr        => 'any',
-  ",
-  resource:       {
-    'action'   => 'deny',
-    'proto'    => 'tcp',
-    'src_addr' => 'any',
-    'dst_addr' => 'any',
+tests[:seq_10_v6] = {
+  desc:           'IPv6 Seq 10',
+  title_pattern:  'ipv6 beaker6 10',
+  platform:       'n(3|7|9)k', # TBD: N5k addrs display as 0001:0001::0001/128
+  manifest_props: {
+    action:   'permit',
+    proto:    'tcp',
+    src_addr: '1:1::1/128',
+    dst_addr: '1:1::2/128',
   },
 }
 
-tests['ipv6_seq_20'] = {
-  desc:           '2.2 ipv6 beaker_6 seq 20',
-  title_pattern:  'ipv6 beaker_6 20',
-  manifest_props: "
-    remark          => 'test remark'
-  ",
-  resource:       {
-    'remark' => 'test remark'
+tests[:seq_20_v4] = {
+  title_pattern:  'ipv4 beaker 20',
+  manifest_props: {
+    action:        'deny',
+    proto:         'tcp',
+    src_addr:      'any',
+    src_port:      'eq 40',
+    dst_addr:      'any',
+    dst_port:      'range 32 56',
+
+    established:   'true',
+    packet_length: 'range 80 1000',
+    precedence:    'flash',
+    redirect:      'Ethernet1/1,Ethernet1/2,port-channel1',
```

Comment:
Yes. Oddly enough it will accept `Ethernet999/99` but it will not nvgen it if it doesn't exist; however, it will nvgen `port-channel` regardless. So I'll just change it to specify port-channels and it should be fine.